core/libs/editor/src/lib/editor-dialog/components/characteristics/characteristic.component.ts

```diff
@@ -29,6 +27,8 @@ import {ChangeDetectorRef, Component, DestroyRef, Input, OnInit, inject} from '@
 import {takeUntilDestroyed} from '@angular/core/rxjs-interop';
 import {FormGroup} from '@angular/forms';
 import {TranslatePipe} from '@ngx-translate/core';
+import {StateCharacteristicComponent} from '../../../../../../../libs/editor/src/lib/editor-dialog/components/characteristics/state-characteristic/state-characteristic.component';
+import {StructuredValueComponent} from '../../../../../../../libs/editor/src/lib/editor-dialog/components/characteristics/structured-value/structured-value.component';
```

Comment:
These 2 imports can be imported like `../../components/characteristics/<component>/<component>.ts`